app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,29 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] I am unable to sign up as a user. When I click the signup button I get this error page. Kindly fix that.
- [ ] After fixing that remember to also ensure that a user is able to reset their password. I was unable to check for this since I couldn't sign up in the first place.

  ![dev-create](https://user-images.githubusercontent.com/36542986/194021596-6d888ba8-207d-48d0-93a7-29f8b0547f5b.jpg)

- [ ] Be sure to also modify your sign-up page to match your wireframe:

  wireframe | your layout
  :---:|:---:
  ![login](https://user-images.githubusercontent.com/36542986/194022668-0b749786-a91a-4e48-9519-4d20d65a14a7.jpg) | ![signup](https://user-images.githubusercontent.com/36542986/194022660-7b4dce5e-d27e-4018-b1ac-5b2729650024.jpg)